osu.Game/Rulesets/Objects/Drawables/DrawableHitObject.cs

```diff
@@ -429,6 +429,13 @@ private void clearExistingStateTransforms()
             base.ClearTransformsAfter(double.MinValue, true);
         }
 
+        /// <summary>
+        /// Removes all previously applied transforms, then reapplies a new set of transforms with potentially different parameters.
+        /// The transforms will use the current <see cref="ArmedState"/>, and they will use the appropriate start times.
+        /// This also takes in account potential overrides defined in <see cref="ApplyCustomUpdateState"/>.
```

Comment:
This comment is quite duplicated imo.

```suggestion
        /// Reapplies the current <see cref="ArmedState"/>.
```